packages/EasyErrorHandler/src/Bridge/Symfony/Builder/ApiPlatformValidationErrorResponseBuilder.php

```diff
@@ -85,6 +105,20 @@ public function buildData(Throwable $throwable, array $data): array
             $matches = [];
 
             switch (1) {
+                case \preg_match(self::MESSAGE_PATTERN_TYPE_ERROR, $throwable->getMessage(), $matches) === 1:
+                    $data['violations'] = [
+                        $matches[1] => [
+                            $matches[3] === 'NULL'
+                                ? (new NotNull())->message
+                                : \sprintf(
```

Comment:
Can we write it inline to be consistent with line 90?

Reply:
Refactored in some another way.